hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -89,10 +89,13 @@ public HiveCatalog(String name, String uri, int clientPoolSize, Configuration co
 
     try {
       List<String> tables = clients.run(client -> client.getAllTables(database));
-      return tables.stream()
+      List<TableIdentifier> tableIdentifiers = tables.stream()
           .map(t -> TableIdentifier.of(namespace, t))
           .collect(Collectors.toList());
 
+      LOG.debug("Listing of namespace [{}] resulted in the following tables: [{}]", namespace, tableIdentifiers);
```

Comment:
What is the purpose of this debug message? I'm not sure how it would help.

Reply:
In Hive we often find that it could be important to know what is the exact information we got from downstream systems, so we can decide if we have to check downstream code to find the problem, or we have to check if we had a bug in Hive code.
For the specific situation (made up example), it could be that HMS returned wrong list of tables because the namespace contained '%' character and HMS used as a wildcard instead of returning exact matches. This debug log could help find the root cause of the issue.